packages/i18n/src/locales/en_US.ts

```diff
@@ -224,14 +226,14 @@ const translations = {
       },
       InsertManyResult: {
         help: {
-          description: 'InsertManyResult Class'
-        },
-        attributes: {
-          acknowledged: {
-            description: 'Acknowledged returned from the server'
-          },
-          insertedIds: {
-            description: 'Array of ObjectIds inserted'
+          description: 'InsertManyResult Class',
+          attributes: {
+            acknowledged: {
+              description: 'Acknowledged returned from the server'
+            },
+            insertedIds: {
+              description: 'Array of ObjectIds inserted'
+            }
```

Comment:
Btw, I was not sure how to actually access the help text from the shell to verify that this was indeed correct, I just inferred that it must be right this way because this is the one where the `attributes` property was on the outer scope.

Reply:
Seems right